treeherder/config/settings.py

```diff
@@ -327,76 +399,73 @@
     },
 }
 
-# rest-framework settings
+# CORS Headers
+CORS_ORIGIN_ALLOW_ALL = True  # allow requests from any host
+
+# Debug Toolbar
+if ENABLE_DEBUG_TOOLBAR:
+    # django-debug-toolbar requires that not only DEBUG be set, but that the request IP
+    # be in Django's INTERNAL_IPS setting. When using Vagrant, requests don't come from localhost:
+    # http://blog.joshcrompton.com/2014/01/how-to-make-django-debug-toolbar-display-when-using-vagrant/
+    # If the Vagrant IPs vary by platform or if there isn't a consistent IP when we switch to Docker,
+    # we'll have to do: https://github.com/jazzband/django-debug-toolbar/pull/805#issuecomment-240976813
+    INTERNAL_IPS = ['127.0.0.1', '10.0.2.2']
+
+if ENABLE_DEBUG_TOOLBAR:
+    INSTALLED_APPS.append('debug_toolbar')
+
+# Elasticsearch
+ELASTICSEARCH_URL = env.str('ELASTICSEARCH_URL', default='')
+
+# Hawkrest
+HAWK_CREDENTIALS_LOOKUP = 'treeherder.webapp.api.auth.hawk_lookup'
+
+# Rest Framework
 REST_FRAMEWORK = {
-    'DEFAULT_PARSER_CLASSES': (
-        'rest_framework.parsers.JSONParser',
-    ),
-    'DEFAULT_FILTER_BACKENDS': (
-        'rest_framework_filters.backends.DjangoFilterBackend',
-    ),
-    'DEFAULT_RENDERER_CLASSES': (
+    'ALLOWED_VERSIONS': ['1.0'],
```

Comment:
This commit changes the d-r-f settings from tuples to lists. Whilst both are valid, the reason we'd used tuples is since that's what the d-r-f docs and defaults use:
https://github.com/encode/django-rest-framework/blob/3.8.2/rest_framework/settings.py#L31